src/plugins/discover/public/application/main/services/discover_state.ts

```diff
@@ -6,11 +6,11 @@
  * Side Public License, v 1.
  */
 
-import { isEqual, cloneDeep } from 'lodash';
+import { cloneDeep, isEqual } from 'lodash';
 import { i18n } from '@kbn/i18n';
 import { History } from 'history';
-import { NotificationsStart, IUiSettingsClient } from '@kbn/core/public';
-import { Filter, FilterStateStore, compareFilters, COMPARE_ALL_OPTIONS } from '@kbn/es-query';
+import { IUiSettingsClient, NotificationsStart } from '@kbn/core/public';
+import { COMPARE_ALL_OPTIONS, compareFilters, Filter, FilterStateStore } from '@kbn/es-query';
 import {
```

Comment:
alright, IDE refactoring, unrelated, but why not, I'd sorting imports by alphabet